inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/store/utils/PulsarUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.audit.store.utils;
+
+import org.apache.pulsar.client.api.Consumer;
+import org.apache.pulsar.client.api.MessageId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PulsarUtils {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PulsarUtils.class);
+    public static void acknowledge(Consumer<byte[]> consumer, MessageId messageId) {
+        try {
+            consumer.acknowledge(messageId);
+        } catch (Exception exception) {
+            LOG.error("Acknowledge has exception ", exception);
```

Comment:
Better to print the topic, subscription and messageId

Reply:
Fixed